samples/corda/fungible-house-token/extension-flows/AssetTransferHelperFlows.kt

```diff
@@ -0,0 +1,246 @@
+package net.corda.samples.tokenizedhouse.flows
+
+import com.weaver.corda.app.interop.states.AssetPledgeState
+import com.weaver.corda.app.interop.flows.GetAssetClaimStatusState
+import com.weaver.corda.app.interop.flows.GetAssetPledgeStatus
+import com.weaver.corda.app.interop.flows.AssetPledgeStateToProtoBytes
+import net.corda.samples.tokenizedhouse.states.FungibleHouseTokenJson
+import com.google.gson.Gson
+import com.google.gson.GsonBuilder
+import net.corda.core.flows.*
+import co.paralleluniverse.fibers.Suspendable
+
+import net.corda.core.identity.Party
+import net.corda.samples.tokenizedhouse.states.FungibleHouseTokenState
+import com.r3.corda.lib.tokens.contracts.states.FungibleToken
+import net.corda.core.node.services.queryBy
+import net.corda.core.identity.CordaX500Name
+import com.r3.corda.lib.tokens.contracts.types.TokenPointer
+import com.r3.corda.lib.tokens.contracts.types.TokenType
+import net.corda.core.contracts.Amount
+import net.corda.core.contracts.ContractState
+import net.corda.core.contracts.StaticPointer
+import com.r3.corda.lib.tokens.contracts.FungibleTokenContract
+import com.r3.corda.lib.tokens.contracts.utilities.issuedBy
+import java.util.Base64
+
+/**
+ * The GetAssetClaimStatusByPledgeId flow fetches the asset claim status in the importing network and returns as byte array.
+ * It is called during the interop query by exporting network before performing the re-claim on asset pledged in exporting network.
+ *
+ * @property pledgeId The unique identifier representing the pledge on an asset for transfer, in the exporting n/w.
+ * @property expiryTimeSecs The time epoch seconds after which re-claim of the asset is allowed.
+ */
+@InitiatingFlow
+@StartableByRPC
+class GetAssetClaimStatusByPledgeId(
+    val pledgeId: String,
+    val expiryTimeSecs: String
+) : FlowLogic<ByteArray>() {
+    /**
+     * The call() method captures the logic to fetch [AssetClaimStatusState] vault state from importing n/w.
+     *
+     * @return Returns ByteArray.
+     */
+
+    @Suspendable
+    override fun call(): ByteArray {
+
+        println("Inside GetAssetClaimStatusByPledgeId(), pledgeId: $pledgeId and expiryTimeSecs: $expiryTimeSecs.")
+
+        println("Creating empty fungible token asset JSON.")
+        var marshalledBlankAssetJson = subFlow(MarshalFungibleToken("dummy-token-type", 0L, "dummy-owner-cert"))
```

Comment:
same comment as in SimpleAssetFlows file, move contents of this subflow to here.